NppMenuSearch/Forms/FlyingSearchForm.cs

```diff
@@ -87,6 +87,15 @@ public void SelectSearchField()
 
         private bool suppressKeyPress;
 
+        private string RemovePrevioustWord(string S, int pos)

```

Comment:
Variable/parameter names should preferably be lowerCamelCase to be consistent wit the rest of the code.

Reply:
This function exists twice. It should instead be a public static function (maybe somewhere else?) to be called from both forms.

Reply:
Done!